llpc/util/llpcShaderModuleHelper.cpp

```diff
@@ -438,12 +441,14 @@ BinaryData ShaderModuleHelper::getShaderCode(const BinaryData &shaderBinary,
 }
 
 // =====================================================================================================================
-// @param shaderBin : Shader binary code
+// @param shaderInfo : Shader module build info
 // @return : The number of bytes need to hold the code for this shader module.
-unsigned ShaderModuleHelper::getCodeSize(const BinaryData &shaderBin) {
-  if (!cl::TrimDebugInfo)
-    return shaderBin.codeSize;
-  return ShaderModuleHelper::trimSpirvDebugInfo(&shaderBin, {});
+unsigned ShaderModuleHelper::getCodeSize(const ShaderModuleBuildInfo *shaderInfo) {
+  const BinaryData& shaderBinary = shaderInfo->shaderBin;
+  bool trimDebugInfo = cl::TrimDebugInfo;
+  if (trimDebugInfo)
```

Comment:
This PR introduced a bug that caused the tests to fail.  It should not have been merged because the tests were not run on it.